examples/kubernetes/cilium-sidecar.yaml

```diff
@@ -0,0 +1,292 @@
+kind: ConfigMap
```

Comment:
Can we wait until this PR is merged https://github.com/cilium/cilium/pull/3125? that way we can move this file to the `plugins` / `addons` directory in `examples/kubernetes`

Reply:
Sure, np.